api/src/main/java/org/apache/unomi/api/rules/Rule.java

```diff
@@ -132,6 +134,15 @@ public boolean isRaiseEventOnlyOnceForProfile() {
         return raiseEventOnlyOnceForProfile;
     }
 
+    /**
+     * Determines whether the event raised when the rule is triggered should only be raised once
+     *
+     * @return {@code true} if the rule-triggered event should only be raised once per profile
+     */
+    public boolean isRaiseEventOnlyOnce() {
+        return raiseEventOnlyOnce;
+    }
+
```

Comment:
A setter is missing, this probably won't work well with JSON deserialization otherwise.